demo-cluster/environments/armo-platform-cluster/adopted/kubernetes-goat/manifest.yaml

```
@@ -0,0 +1,992 @@
+  ipFamilyPolicy: SingleStack
+  ports:
+  - nodePort: 30001
+    port: 8080
+    protocol: TCP
+    targetPort: 8080
+  selector:
+    app: vulnerable-app
+  sessionAffinity: None
+  type: NodePort
+# --- ServiceAccount/kubescape-sizing-checker ---
+apiVersion: v1
```

YAMLlint (1.37.1)

[error] duplication of key "apiVersion" in mapping

(key-duplicates)